src/inputmethodkit.cs

```diff
@@ -150,7 +151,7 @@ partial interface IMKInputController : IMKStateSetting, IMKMouseHandling {
 		NSRange ReplacementRange { get; }
 
 		[Export ("markForStyle:atRange:")]
-		NSDictionary Mark (nint style, NSRange range);
+		NSDictionary GetMark (nint style, NSRange range);
```

Comment:
This one is not: https://developer.apple.com/reference/inputmethodkit/imkinputcontroller/1385381-markforstyle?language=objc